### DISCUSSION TITLE
如何直接显示单元格里的公式值，而不是计算结果

### DISCUSSION THREAD

**Discussion by sea-yu** [dream-num/univer#3767] [Q&A | 常见问题]
你好，我想要让单元里面，显示公式的内容而不是公式的计算结果呢？比如一个单元格 `=SUM(1, 1)` 我想要这个单元格就显示 `=SUM(1, 1)` 而不是2，并且我不知道用户要在哪个单元格写公式，所以目前我不能指定单元格让它的t的变成文本。

ps: 需要保留公式提示功能。

**Comment by jikkai** [comment]
Duplicate of #3236
